cmd/keepalive/keepalive.go

```diff
@@ -0,0 +1,109 @@
+// Example: client session keep alive
+// By default, the token expires in 4 hours if the connection is idle. CLIENT_SESSION_KEEP_ALIVE parameter will
+// have a heartbeat in the backend to keep the connection alive explicit heartbeats.
```

Comment:
have a heartbeat in the background to keep the connection alive by making explicit heartbeats

Reply:
Fixed.